codejam-canvas/script.js

```diff
@@ -0,0 +1,47 @@
+const canvas = document.querySelector('#canvas');
+const ctx = canvas.getContext('2d');
+let inputs = document.querySelectorAll('input[name="size"]');
+for (let i = 0; i < inputs.length; i++) {
+    inputs[i].addEventListener('change', function () {
+        ctx.clearRect(0, 0, canvas.width, canvas.height);
+        getArr(this.value);
+    });
+}
+
+function getArr(scale) {
+    let url = '';
+    if (scale == '4') {
+        url = 'https://raw.githubusercontent.com/rolling-scopes-school/tasks/master/tasks/stage-2/codejam-canvas/data/4x4.json';
+    } else if (scale == '32') {
+        url = 'https://raw.githubusercontent.com/rolling-scopes-school/tasks/master/tasks/stage-2/codejam-canvas/data/32x32.json';
+    } else {
+        let img = document.createElement('img');
+        img.crossOrigin = "Anonymous";
+        img.src = 'https://raw.githubusercontent.com/rolling-scopes-school/tasks/master/tasks/stage-2/codejam-canvas/data/image.png';
+        img.onload = function() {
+            ctx.drawImage(img, 0, 0, canvas.width, canvas.height);
+            return;
```

Comment:
this return statement is useless since on the end of the function execution it will exit implicitly